app/models/reputation.rb

```diff
@@ -0,0 +1,5 @@
+class Reputation
+  def self.calculate(object)
```

Comment:
Unused method argument - `object`. If it's necessary, use `_` or `_object` as an argument name to indicate that it won't be used. You can also write as `calculate(*)` if you want the method to accept any arguments but don't care about them.